_posts/2019-02-25-the-western-grid.md

```diff
@@ -0,0 +1,136 @@
+
+The early American Congress further solidified the acre as the official unit of land measurement with the Land Ordinance of 1785 and the resulting North West Ordinance of 1787. These ordinances provided the framework of the public lands and the procedures for organizing territorial lands west of the Application Mountains.
+
+## Public Land Survey System
+{: .text-left}
+
+I know, I know… that was probably more history than you were looking for, but really, that’s the annotated, crash-course for understanding the basis of the present-day Public Land Survey System (PLSS) (Fig 4). This system was originally proposed by Thomas Jefferson, and was mandated by Congress to oversee the cadastral surveys of the public lands. Essentially, it was set up to facilitate the transfer of Federal lands to private citizens. The Bureau of Land Management (BLM) is the official record keeper of the surveys. Over the past 200 years, almost 1.5 billion acres have been surveyed into townships (and ranges) and sections. These surveys were conducted primarily west of the original thirteen colonies and north of Texas (Texas has Spanish roots and Spanish land grants).  Presently, areas of Alaska are still being surveyed.
+
+<figure class="caption caption--left caption--flow">
+  <a href="{{ "/images/principal_meridians.png" | prepend: site.baseurl }}" title="click to see the full sized image">
+    <img src="{{ "/images/principal_meridians_sm.png" | prepend: site.baseurl }}" alt="PLSS Principal Meridians">
+  </a>
+  <figcaption class="caption__text">Fig 4: PLSS Principal Meridians</figcaption>
+</figure>
+<figure class="caption caption--right caption--flow">
+  <img src="{{ "/images/salt_lake_meridian.png" | prepend: site.baseurl }}" alt="Salt Lake Meridian" />
+  <figcaption class="caption__text">Fig 5: Salt Lake Meridian</figcaption>
+</figure>
+
+As with any measuring system, you need a starting point. Within the PLSS, this is called the Principal Meridian. In Utah, this location is in Salt Lake City at Main Street and South Temple (Fig 5). All official surveys in Utah have an underlying reference to this point. In other words, every Township and Section in Utah is either North, South, East, or West of this location. Other PLSS states have their own unique principal meridians (Fig 4).
+
+The Townships are made up of six square miles, with each square mile known as a Section. These Sections are eventually subdivided down to an acre by way of Half Sections, Quarter Sections, Quarter-Quarter Sections, etc. (Fig 6). An acre, again, is simply 10 square chains - and there are 640 acres in a Section (a square-mile) or 40 acres in a quarter-quarter section. Hence the phrase, “40 acres and mule” is all you needed to be self-sufficient.
```

Comment:
Footnote 6 (For Figure 6): 
“US Public Land Survey System,” (image), San Francisco Estuary Institute & The Aquatic Science Center, accessed March 01, 2019. https://www.sfei.org/it/gis/map-interpretation/projections-and-survey-systems#sthash.D2D94lF3.sWlKZBU3.dpbs.